packages/itwinui-react/src/core/DropdownMenu/DropdownMenu.tsx

```diff
@@ -30,6 +30,17 @@ export type DropdownMenuProps = {
    * Child element to wrap dropdown with.
    */
   children: React.ReactNode;
+  /**
+   * Middleware options.
+   *
+   * By default, `hide` is enabled. If the menu gets hidden even when it shouldn't (e.g. some custom styles interfering
+   * with the trigger's hide detection) consider disabling the `hide` middleware.
+   *
+   * @see https://floating-ui.com/docs/middleware
+   */
+  middleware?: {
+    hide?: boolean;
```

Comment:
It should be under `dropdownMenuProps` in `ComboBox` and under `popoverProps` in `Select`.

Reply:
> Changed ([92a8517](https://github.com/iTwin/iTwinUI/commit/92a851724ec51843141f0ac73381bde1ce3be122)).

Just a nit: we shouldn't rely on `DropdownMenuProps` in `ComboBox`; we should use `usePopover` props instead. The `ComboBox` has nothing to do with the `DropdownMenu` (yes, the naming of `dropdownMenuProps` is unfortunate, but we cannot change that now).

> Do we want to expose props to pass down to `Menu`, `DropdownMenu`, etc.?

Yeah let's do that at least for `SplitButton`. For `HeaderButton` we can leave it out, since it's always in one spot (and the component is built in a questionable way and should be replaced in the future).

Reply:
Maybe `dropdownMenuProps` instead of `menuProps`?

Reply:
Another option is `popoverProps` which I guess would be more appropriate. I don't feel strongly either way, so I'd say use your best judgement.